rwatch/ui/window.c

```diff
@@ -153,6 +191,11 @@ uint16_t window_count(void)
     uint16_t count = 0;
     Window *w;
     
+    if (appmanager_get_thread_type() == AppThreadOverlay)
```

Comment:
same

Reply:
Though, really, maybe we should just have these functions do the right thing automatically?  Then you always just call window_x, and you get the "right" answer depending on what thread you're in.

Reply:
these functions are special exceptions because they have no concept of which overlay is being presented. The idea being that overlay is an internal thing, we should kick the developer to try and use the more specialised form of window_count(overlay context).
It is marginally over engineered in that we have more overlay support than is absolutely required, but it allows me to add a pet feature set so we can stack notifications (reading an incoming message AND overlaying a small battery low icon). All is designed without going full screen for overlays.